README.md

```diff
@@ -46,29 +73,74 @@ console.log("PB = ", keys.public_key("Bob"));
 keys.free();
 ```
 
-# Benchmarks
+### Building the C FFI module
+
+To build the C bindings, you can run
+
+    make ffi
+
+To build the release version (recommended):
+
+    make ffi-release
+
+To run the small demo:
+
+    make demo
+    ./bin/demo
+
+## Benchmarks
 
 To run the benchmarks:
 
     $ cargo bench
 
 The benchmarks use Criterion and will produce nice graphs (if you have gnuplot installed)
 
-To run the benchmarks with SIMD instructions:
+## Hardware wallet support
 
-    $ cargo bench --features "avx2"
+Tari Crypto supports hardware wallets like Ledger Nano S, X and S Plus when included in the project with the "no_std" 
+and "borsh_ser" feature flags. Whenever `tari_rypto` is changed, test locally if the proper "no_std" is adhered to by 
+running the following commands: 
+```
+rustup target add --toolchain nightly thumbv8m.main-none-eabi
+export PATH="$HOME/.cargo/bin:$PATH"
+cargo +nightly rustc --release --crate-type lib --no-default-features --features "no_std" --features "borsh_ser" -Zavoid-dev-deps --target=thumbv8m.main-none-eabi
+```
 
-# Building the C FFI module
+The following project can also be used to test if everything works end-to-end: https://github.com/tari-project/ledger.
 
-To build the C bindings, you can run
+## Feature Flags
 
-    make ffi
+Some of the utilities can be removed with feature flags. The following feature flags are available:
 
-To build the release version (recommended):
+### no_std
 
-    make ffi-release
+This feature removes the dependency on the Rust standard library. This is useful for embedded systems and hardware.
 
-To run the small demo:
+### borsh_ser
 
-    make demo
-    ./bin/demo
+This will include support for borsh serialization.
+
+### wasm
+
+This will include support for WebAssembly.
+
+### ffi
+
+This will include support for the C FFI module.
+
+### zero
+
+This will include support for zeroing memory.
```

Comment:
I think this could be expanded and should add a warning about not enabling it. I would also mention that it uses the zeroize crate